addons/mail/static/src/components/chat_window/chat_window.js

```diff
@@ -69,6 +69,9 @@ export class ChatWindow extends Component {
      * @private
      */
     _applyVisibleOffset() {
+        if (!this.root.el) {
+            return;
+        }
```

Comment:
Condition should possibly be on parent here? I would expect method not to be called when its wrong to call it.

Reply:
Changes done. It's a bit weird because we don't get why this is guarded.
If this method no longer uses `this.root.el`, it's wrong to guard `this.root.el` above...

Reply:
It feels strange because the condition is written with technical values, but generally speaking it makes sense:

"if the component doesn't exist -> there is nothing to do (regardless of whatever or however that is done)"